config/routes.rb

```diff
@@ -0,0 +1,61 @@
+
+
+Rails.application.routes.draw do
+  root 'media_ranker#index'
+
+  resources :media_ranker
+  resources :albums
+  # post 'albums/create' => 'albums#create'
+  patch 'albums/:id/upvote' => 'albums#upvote'
+
+  resources :books
+  # post 'books/create' => 'books#create'
+  patch 'books/:id/upvote' => 'books#upvote'
+
+  resources :movies
+  patch 'movies/:id/upvote' => 'movies#upvote'
```

Comment:
We can actually combine these two lines and allow the `upvote` action for each resource to benefit from its resource definition:
```ruby
resources :movies do
  member do
    patch 'upvote'
  end
end
```